core-bundle/src/Resources/contao/drivers/DC_Folder.php

```diff
@@ -2382,6 +2382,10 @@ public function sync()
 				case 'Deleted':
 					$arrMessages[] = '<p class="tl_error">' . sprintf($GLOBALS['TL_LANG']['tl_files']['syncDeleted'], \StringUtil::specialchars($file)) . '</p>';
 					break;
+
+				default:
+					$arrMessages[] = '<p class="tl_error">' . \StringUtil::specialchars($buffer) . '</p>';
```

Comment:
```suggestion
					$arrMessages[] = '<p class="tl_error">' . StringUtil::specialchars($buffer) . '</p>';
```

This way it is upstream compatible. I am always doing it like this to prevent merge conflicts.